src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -106,7 +106,7 @@ export default class FilterManager extends WebGLManager
 
             // TODO we should fit the rect around the transform..
         }
-        else
+        else if (filters[0].fitToDestination)
```

Comment:
It wouldn't be ideal for performance, but should we check all the filters that are getting added, in case there are multiple and one does not want to be fitted?